pkgs/tools/misc/ttygif/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "ttygif";
+  version = "1.4.0";
+
+  src = fetchurl {
+    url = "https://github.com/icholy/ttygif/archive/1.4.0.tar.gz";
+    sha256 = "0apf7j6291rgv4s7c58cjq284ldc3scsws0003wdqayyr9fxr8vc";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ttygif $out/bin
+  '';
```

use the pre-existing Makefile, replace installPhase with
```suggestion
  makeFlags = [ "PREFIX=${placeholder "out"}" ];
```

---

then the default `make install` command will work as intended.